model.js

```diff
@@ -48,4 +48,22 @@ exports.fetchCommentsByReviewsId = (inputId) => {
   });
 };
 
+exports.addCommentsByReviewsId = (inputId, username, body) => {
+  if (inputId && username && body) {
+    const queryString = `INSERT INTO comments (review_id, author, body) VALUES ($1, $2, $3) RETURNING *;`;
+    return db
+      .query(queryString, [inputId, username, body])
+      .then((commentData) => {
+        return commentData.rows[0];
+      })
+      .catch((error) => {
+        if (error.code === "23503") {
+          return Promise.reject({ status: 404, msg: "Path not found" });
+        }
```

Comment:
if the error code is not 23503 it will just swallow this error and potentially send back a 201 in the controller, in an else you should either throw the error again for the next catch layer or Promise.reject something to handle that situation